final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,52 @@
+import feed
+import feedparser
+import ssl
+import logging
+from args_creater import arguments
+
+
+if hasattr(ssl, '_create_unverified_context'):
+    ssl._create_default_https_context = ssl._create_unverified_context
+
+
+def main():
+    """main function"""
+    args = arguments()
+
+    rss = args.source
+
+    logging_level = 'WARNING'
+    if args.verbose:
+        logging_level = 'INFO'
+
+    logging.basicConfig(level=logging_level,)
+    logging.info('Started')
+
+    parsed = feedparser.parse(rss)
+    if 'bozo_exception' in parsed.keys():
```

Comment:
check `bozo` argument in the `parsed` object. Probably you will find that it is more convenient to use it for your check

Reply:
Fixed